app/views/assignment_repos/_unlinked_user.html.erb

```diff
@@ -0,0 +1,33 @@
+<% view = AssignmentRepoView::ShowView.new(assignment_repo: assignment_repo) %>
+
+
+<div class='d-flex flex-row flex-justify-between flex-items-center assignment-repo-list-item'>
+  <div class='d-flex'>
+    <div>
+      <%= link_to unlinked_user.github_user.html_url, class: "tooltipped tooltipped-n", target: '_blank', 'aria-label' => unlinked_user.github_user.login do %>
+        <%= image_tag unlinked_user.github_user.github_avatar_url(96), height: 48, width: 48, class: 'avatar avatar-large', alt: "@#{unlinked_user.github_user.login}" %>
```

Comment:
Totally agree. I want to do the group assignment view after this, then spend some time seeing what we can extract into shared views or view helpers. :+1: